pkg/cassandra/session.go

```diff
@@ -54,6 +54,7 @@ type Session interface {
 type UpdateQuery interface {
 	Exec() error
 	String() string
+	ScanCAS(dest ...interface{}) (bool, error)
```

Comment:
I suggest adding the comment from gocql, since the semantics of this method are not obvious
```
// ScanCAS executes a lightweight transaction (i.e. an UPDATE or INSERT
// statement containing an IF clause). If the transaction fails because
// the existing values did not match, the previous values will be stored
// in dest.
```